101-vsts-cloudloadtest-rig/azuredeploy.json

```diff
@@ -0,0 +1,226 @@
+﻿{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "vstsAccountName":{
+         "type":"string",
+         "metadata":{
+            "description": "Please enter the VSTS account name. If you access your VSTS account using 'https://yourAccountName.visualstudio.com', enter yourAccountName."
+         }
+      },
+      "PAT":{
+         "type":"securestring",
+         "metadata":{
+            "description": "Personal Access Token (PAT) for the VSTS account. You should select the  scope as 'Load test (read and write)'. Please refer 'https://www.visualstudio.com/en-us/get-started/setup/use-personal-access-tokens-to-authenticate' for more details."
+         }
+      },
+      "agentCount":{
+         "type":"int",
+         "metadata":{
+            "description": "Number of load generating agent machines to provision"
+         },
+         "defaultValue":1
+      },
+      "adminUsername":{
+         "type":"string",
+         "metadata":{
+            "description":"Username for the virtual machine."
+         }
+      },
+      "adminPassword":{
+         "type":"securestring",
+         "metadata":{
+            "description":"Password for the virtual machine."
+         }
+      }
+   },
+   "variables":{
+      "sizeOfDiskInGB":"100",
+      "imagePublisher":"MicrosoftWindowsServer",
+      "imageOffer":"WindowsServer",
+      "addressPrefix":"10.0.0.0/16",
+      "sequenceVersion":"[uniqueString(resourceGroup().id)]",
```

Comment:
Using just the resource group id for uniquestring seed is going to cause failures for any subsequent deployments of this template into the same resource group.